src/pages/RegisterPage.tsx

```diff
@@ -77,35 +96,99 @@ const RegisterPage = () => {
     setValid(v => ({ ...v, [target]: value }))
   }, [])
 
+  const handleNextPage = () => {
+    if (page === 1) {
+      emailForm.handleSubmit(() => setPage(2))()
+    }
+    if (page === 2) {
+      userInfoFrom.handleSubmit(() => setPage(3))()
+    }
+    if (page === 3) {
+      credentialForm.handleSubmit(onSubmit)()
+    }
+  }
+
```

Comment:
```suggestion
  useEffect(() => {
    if (isAuth) navigate('/mypage')
  }, [isAuth, navigate])
```

로그인 페이지와 마찬가지로, 이미 로그인이 되어있다면 마이페이지로 리다이렉트 시키는 것이 좋을 것 같읍니다